analyzers/tests/SonarAnalyzer.UnitTest/TestCases/VariableShadowsField.CSharp9.cs

```diff
@@ -32,8 +32,8 @@ public void PositionalPattern()
         _ = (1, 2) is (_, _) bar; // Noncompliant
     }
 
-    public void PatternDesignation()
+    public void VarPattern()
     {
-        _ = (1, 2) is (var bar, _); // Noncompliant
```

Comment:
review note: this case is covered in [VariableShadowsField.CSharp10.cs ](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/VariableShadowsField.CSharp10.cs)